lib/isafw/isafw.py

```diff
@@ -30,7 +30,7 @@
 
 import sys
 import traceback
-import isafw.isaplugins as isaplugins
+import isaplugins
```

Comment:
This will probably break Python3 again.

If there's no import statement that works for both, then we just have to try the two options:
```
try:
    # absolute import for Python3
    import isafw.isaplugins as isaplugins
except ImportError:
    # relative import for Python2 and when not using bitbake
    import isaplugins
```

With that, the "import absolute_import" line can be removed again.